apps/api/package.json

```diff
@@ -5,9 +5,8 @@
 	"type": "module",
 	"main": "src/main.ts",
 	"scripts": {
-		"db-test": "npx tsx --env-file=.env ./src/db/index.ts",
 		"dev": "npx tsx watch --env-file=.env ./src/main.ts",
-		"test": "npx tsx --test --test-reporter=spec --experimental-test-coverage ./src/tests/*"
+		"test": "npx tsx --env-file=.env --test --test-reporter=spec --experimental-test-coverage ./src/tests/**/*"
```

Comment:
We could potentially set up testcontainers now and replace `.env` here with `.env.test`